### PR TITLE
Initial functioning deployment

### PR DESCRIPTION
**Pull request by iw-gtadam**

### CODE REVIEW

File: config/azkaban/exec-server/azkaban.properties [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Azkaban Personalization Settings
+azkaban.name=Test
+azkaban.label=My Local Azkaban
+azkaban.color=#FF3601
+azkaban.default.servlet.path=/index
+web.resource.dir=web/
+default.timezone.id=America/Los_Angeles
```
Comment (IanUtley) [inline config/azkaban/exec-server/azkaban.properties:7]: Can we change to Europe/London ?

File: config/azkaban/web-server/azkaban.properties [hunk 2]
```diff
@@ -0,0 +1,36 @@
+azkaban.name=Test
+azkaban.label=My Local Azkaban
+azkaban.color=#FF3601
+azkaban.default.servlet.path=/index
+web.resource.dir=/azkaban-web-server/web
+default.timezone.id=America/Los_Angeles
```
Comment (IanUtley) [inline config/azkaban/web-server/azkaban.properties:6]: Ditto. Europe/London

File: config/azkaban/exec-server/azkaban.properties (hunk 1 above)
Comment (iw-gtadam) [reply config/azkaban/exec-server/azkaban.properties]: Amended

File: config/azkaban/web-server/azkaban.properties (hunk 2 above)
Comment (iw-gtadam) [reply config/azkaban/web-server/azkaban.properties]: Amended

**Review by IanUtley — APPROVED** [review]

**Pull request merged**